Symfony/CS/Tests/Fixer/AbstractFixerTestBase.php

```diff
@@ -18,11 +18,16 @@
  */
 abstract class AbstractFixerTestBase extends \PHPUnit_Framework_TestCase
 {
+    protected function getFixerFullName()
+    {
+        return 'Symfony\CS\Fixer'.substr(get_called_class(), strlen(__NAMESPACE__), -strlen('Test'));
+    }
+
     protected function getFixer()
     {
-        $fixerName = 'Symfony\CS\Fixer'.substr(get_called_class(), strlen(__NAMESPACE__), -strlen('Test'));
+        $name = $this->getFixerFullName();
```

Comment:
`getFixerFullName` was created to be used in tests as well (https://github.com/keradus/PHP-CS-Fixer/commit/e3181f94a38150b5d765c37882a4ac13706219f6), after latest comment this could be reverted. good point.